config/application.rb

```diff
@@ -67,14 +67,16 @@ class Application < Rails::Application
     config.contentful_management_access_token = ENV.fetch('CONTENTFUL_MANAGEMENT_TOKEN', credentials.dig(:contentful, :management_access_token)) # TODO: use service account management token
     config.contentful_environment             = ENV.fetch('CONTENTFUL_ENVIRONMENT', credentials.dig(:contentful, :environment))
 
+    config.active_record.schema_format = :sql
```

Comment:
This will have affected the review app and made the ruby schema essentially redundant

Reply:
No, I thought I had it fixed, but it popped up again whenever I run `bin/docker-rspec`